### ISSUE TITLE
Impossible to expose Root note as CV port in Carla

### ISSUE THREAD

**Issue by Amulet-git** [sjaehn/BSEQuencer#32]
while other discrete parameters such as Octave can. Can it be as simple as changing the type of parameter of Root note from list to discrete integer?

**Comment by sjaehn** [comment]
This is right. The root note is realized by two parameters (or to be precise: two lv2 ports): `root` and `signature`. Both are enumerations. `root` has got the range from 0 to 11 (from note C to B), but skips the values for the "signed" notes. `signature` has got values from -1 to 1. Thus, the sum `root + signature` is the number of semitones starting from C.

And indeed, enumerations can't be exported as CV in Carla. Exporting floats or integers from control ports to CV is a great feature of Carla. You may discuss with @falkTX if an export of enumerations makes sense too (or not).

Another option would be to merge `root` and `signature` to a single parameter. However, this would break the backward compatibility and can't be done for this plugin version. But it is a good idea for a B.SEQuencer2.
